py_gtfs_rt_ingestion/lib/s3_utils.py

```diff
@@ -85,65 +85,47 @@ def strip_timestamp(fileobject: str) -> str:
         return []
 
 
-def invoke_async_lambda(function_arn: str, event: dict) -> None:
+def _move_s3_object(filename: str, to_bucket: str) -> None:
     """
-    Invoke a lambda method asynchronously.
-
-    :param function_arn: Lambda function's Amazon Resource Name
-    :param event: the event information passed into the invocation of the lambda
-                  function
-
-    No return value
-    """
-    lambda_client = boto3.client("lambda")
-    lambda_client.invoke(
-        FunctionName=function_arn,
-        InvocationType="Event",
-        Payload=json.dumps(event),
-    )
-
-
-def _move_s3_object(filename: str, destination: str) -> None:
-    """
-    move a single s3 file to the destination bucket. break the incoming s3 path
-    intco parts that are used for copying. each execution will spin up a session
-    and get a resource from that session to avoid threading issues.
+    move a single s3 file to the to_bucket bucket. break the incoming s3 path
+    into parts that are used for copying. each process will have it's own
+    boto session and resource available, from the _init_process_session function
+    to avoid multi-processing issues
 
     :param filename - expected as 's3://my_bucket/the/path/to/the/file.json
-    :param destination bucket name
+    :param to_bucket bucket name
     """
     process_logger = ProcessLogger(
         "move_s3_object",
         filename=filename,
-        destination=destination,
+        to_bucket=to_bucket,
     )
     process_logger.log_start()
 
     try:
-        session = boto3.session.Session()
-        s3_resource = session.resource("s3")
+        s3_resource = current_process().__dict__["boto_s3_resource"]
```

Comment:
pull `s3_resource` from process dictionary, instead of having the create one for every file move